application/src/main/java/org/opentripplanner/transit/service/TimetableRepository.java

```diff
@@ -286,9 +292,15 @@ public void addFeedInfo(FeedInfo info) {
    * Ideally we would want to interpret times in the time zone of the geographic location where the
    * origin/destination vertex or board/alight event is located. This may become necessary when we
    * start making graphs with long distance train, boat, or air services.
+   * <p>
+   * Defaults to GMT if not set and there are no agencies specified.
    */
   public ZoneId getTimeZone() {
-    return timeZone;
+    if (timeZone != null) {
+      return timeZone;
+    }
+    LOG.warn("graph contains no agencies (yet); API request times will be interpreted as GMT.");
+    return ZoneId.of("GMT");
```

Comment:
Consider changing this to "init once", is it ok to use 2 different time-zones? It is hard for me to tell if the code is changed here or not.